frontend/src/metabase/notifications/AddEditSidebar/DefaultParametersSection.jsx

```diff
@@ -12,8 +13,27 @@ import Heading from "./Heading";
 function formatDefaultParamValues(parameters) {
   return parameters
     .map((parameter) => {
-      const value = conjunct([].concat(parameter.default), t`and`);
-      return value && `${parameter.name} is ${value}`;
+      const { name, type, default: defaultValue } = parameter;
+
+      if (!defaultValue) {
+        return null;
+      }
+
+      let formattedValue;
+      if (type.startsWith("date/")) {
+        const values = [].concat(defaultValue);
+        const formattedValues = values
+          .map((val) => formatDateValue(parameter, val))
+          .filter(Boolean);
+
+        if (formattedValues.length > 0) {
+          formattedValue = conjunct(formattedValues, t`and`);
+        }
+      } else {
+        formattedValue = conjunct([].concat(defaultValue), t`and`);
+      }
+
+      return formattedValue && `${name} -> ${formattedValue}`;
```

Comment:
Nitpick but I'd use a good old if statement here, especially since this is JS and not TS, in order to avoid unexpected returned values in case formattedValue isn't a string or whatever.